src/test/java/ibm/jceplus/junit/base/BaseTestAESGCMCipherInputStreamExceptions.java

```diff
@@ -58,51 +60,26 @@ private void do_gcm_AEADBadTag() throws Exception {
         // Create stream for decryption
         CipherInputStream in = getStream("GCM", ct);
 
-        try {
-            //System.out.println ("trying to read");
-            in.read(read);
-            //System.out.println ("Warning: The test is a failure till CipherInputStream is fixed.");
-            // throw new RuntimeException("Fail: CipherInputStream.read() " +
-            //       "returned " + size + " and didn't throw an exception.");
-        } catch (IOException e) {
-            //.System.out.println ("IOException while reading");
-            Throwable ec = e.getCause();
-            if (ec instanceof AEADBadTagException) {
-                System.out.println("  Pass.");
-            } else {
-                System.out.println("  Fail: " + ec.getMessage());
-                throw new RuntimeException(ec);
-            }
-        } finally {
-
-            try {
-                in.close();
-            } catch (IOException e) {
-                Throwable ec = e.getCause();
-                if (ec instanceof AEADBadTagException) {
-                    System.out.println("  Pass.");
-                } else {
-                    System.out.println("  Fail: " + ec.getMessage());
-                    throw new RuntimeException(ec);
-                }
-            }
-
+        int amountRead = in.read(read);
 
-        }
+        // Expect 96 bytes since this is closest block size we can read without going over.
+        assertEquals(96, amountRead);
     }
 
-    /* Short read stream,
+    /**
+     * Short read stream.
+     * 
      * This test
      *   1) Encrypt 100 bytes with AES/GCM/NoPadding
```

Comment:
Should be encrypt 600 bytes with AES/GCM/NoPadding

Reply:
Yes updated this description to be 600 bytes not 100.